cmd/resim/commands/test_suites.go

```diff
@@ -164,7 +164,7 @@ func init() {
 	runTestSuiteCmd.Flags().String(testSuiteBuildIDKey, "", "The ID of the build to use in this test suite run.")
 	runTestSuiteCmd.MarkFlagRequired(testSuiteBuildIDKey)
 	// Parameters
-	runTestSuiteCmd.Flags().StringSlice(testSuiteParameterKey, []string{}, "(Optional) Parameter overrides to pass to the build. Format: <parameter-name>:<parameter-value>. Accepts repeated parameters or comma-separated parameters.")
+	runTestSuiteCmd.Flags().StringSlice(testSuiteParameterKey, []string{}, "(Optional) Parameter overrides to pass to the build. Format: <parameter-name>=<parameter-value> or <parameter-name>:<parameter-value>. The equals sign (=) is recommended, especially if parameter names contain colons. Accepts repeated parameters or comma-separated parameters.")
```

Comment:
I know you didn't add the last sentence here but an example might be useful ("e.g. parameter-1=value,parameter-2=value")